app/src/assets/css/Selectable.css

```diff
@@ -10,6 +10,9 @@
   flex-direction: column;
   border-radius: 2px;
   transition: 0.25s border-color;
+  user-select: none;
+  cursor: pointer;
+  cursor: hand;
```

Comment:
This looks a really outdated CSS hack.
Pretty sure all browsers  from the last decade support pointer.

Personally I'm not a fan of adding `pointer` to all clickable elements. I mainly use them for hyperlinks. 
https://ux.stackexchange.com/a/105027